.github/workflows/llvm-quick-fuzz.yml

```diff
@@ -62,7 +80,16 @@ jobs:
       - shell: bash
         name: Run fuzzing
         run: |
-          cabal exec -ffuzz fuzz-llvm-disasm -- -n 50 --junit-xml results.xml -o results --collapse -I /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include --clang-flags="-O -w" --clang-flags="-O -w -g" -c clangy
+          cabal exec -ffuzz fuzz-llvm-disasm -- -n 50 --junit-xml results.xml -o results --collapse --clang-flags="-O -w" --clang-flags="-O -w -g" -c clangy
```

Comment:
Not something you changed, but any idea why `--clang-flags` is duplicated/updated here?

Reply:
Wow. Wooooooow. Removing the (seemingly) duplicate flags _does_ [cause it to fail](https://github.com/GaloisInc/llvm-pretty-bc-parser/actions/runs/4440936039). What's more, the output does _not_ make it obvious what goes wrong. Sigh.

Something smells about this, but this PR has already dragged on long enough. I'll just continue to stick with the status quo of `--clang-flags="-O -w" --clang-flags="-O -w -g"` for now.

Reply:
Yikes!  I agree: we should probably have a better syntax/requirement spec here, but it definitely seems involved and beyond the scope of this PR.  Thanks for checking though!